metadata/simple-tile.xml

```diff
@@ -18,42 +18,42 @@
 		<option name="button_move" type="button">
 			<_short>Button move</_short>
 			<_long>When the specified button is held down, you can drag and drop tiled windows to reorder them and to change the tiling direction.</_long>
-			<default>&lt;super&gt; BTN_LEFT</default>
```

Comment:
I don't see a reason to set the simple-tile bindings to none.

Reply:
`simple-tile` is not enabled by default and these bindings will likely conflict with the user config.

Reply:
The plugins are designed in such a way that this is fine. I intentionally made both bindings the same: simple-tile will be used only for tiled windows, and the move binding will be used only for floating windows.